packages/expo-ui/android/src/main/java/expo/modules/ui/TextInputView.kt

```diff
@@ -47,6 +49,17 @@ fun String.keyboardType(): KeyboardType {
   }
 }
 
+fun String.autoCapitalize(): KeyboardCapitalization {
```

Comment:
Can you also mark `fun String.keyboardType(): KeyboardType` as private while you're at it 😄 
```suggestion
private fun String.autoCapitalize(): KeyboardCapitalization {
```